src/osyris/spatial/__init__.py

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Osyris contributors (https://github.com/osyris-project/osyris)
+
+# flake8: noqa
+
+from .subdomain import extract_sphere, extract_box
```

Comment:
Do we also need to import `spatial` in the top-level `__init__.py` for the whole of `osyris`?
Basically, without it, can we do
```Py
import osyris
osyris.spatial.extract_sphere(data)
```
or do we have to manually import the spatial module? i.e.
```Py
import osyris
from osyris import spatial
spatial.extract_sphere(data)
```

Reply:
Since the functions in `spatial` are imported in osyris's `__init__`, we can simply do `osyris.extract_sphere` (same as maps and histograms). You can also do `import osyris.spatial as sp`.

Reply:
I just tried, can't seem to get the spatial module without an import in the top level `__ini__`